### ISSUE TITLE
Advanced snapshot deployments for devices

### ISSUE THREAD

**Issue by akashtalole** [flowforge/flowforge#1323]
### Description

As a user

-  Shall create snapshot for specific flow
- Shall add tags for devices
- Then deploy specific snapshot to mentioned device tag

**Comment by joepavitt** [comment]
Hi @akashtalole, thanks for raising this issue. I want to understand a bit more about your request.

When you're asking about a "specific flow", is this a flow you would already have running in FlowForge?

Regarding the devices deployment, are you implying that you would not want to deploy a given snapshot too _all_ devices in a project, but a select few (which you are proposing to define via "device tags")?

**Comment by akashtalole** [comment]
@joepavitt Right. Flows are created on Flowforge project. Consider I have Flow1, Flow2, Flow3, ... Flow n and I have suppose 100 devices for different iot use cases. I need to deploy Flow1 to first 10 devices, Flow2 to next 50 devices and so on. 

In current implementation, snapshot target(all flows) are deployed to all added devices in project. 


If we can define device tags and specify separate snapshot target for device tags then it is possible if we can create flow based snapshots.
